output.go

```diff
@@ -829,9 +829,10 @@ func OutputsSyntacticalFoundry() OutputsSyntacticalValidationFunc {
 			return fmt.Errorf("%w: output %d, less than equal zero", ErrFoundryOutputInvalidMaximumSupply, index)
 		}
 
-		if r := foundryOutput.CirculatingSupply.Cmp(foundryOutput.MaximumSupply); r == 1 {
-			return fmt.Errorf("%w: output %d, bigger than maximum supply", ErrFoundryOutputInvalidCirculatingSupply, index)
-		}
+		// TODO: more syntactic checks are possible but not defined in the TIP:
+		// - minted - melted > 0: foundry can never have melted more than minted
+		// - minted - melted <= max supply: can never have minted more than max supply
+		// - melted <= minted: can never have melted more than minted
```

Comment:
Where are the syntactic checks from the TIP?
 - Minted Tokens must not be greater than Maximum Supply.
 - Melted Tokens must not be greater than Minted Tokens.